File: wss_test.py
import asyncio
import websockets
import json

WSS_URL = "wss://ws.7k2v9x1r0z8t4m3n5p7w.com"

async def test_connection():
    print(f"Connecting to {WSS_URL}...")
    try:
        async with websockets.connect(WSS_URL) as websocket:
            print("Connected!")
            
            # Wait for initial message
            try:
                msg = await asyncio.wait_for(websocket.recv(), timeout=10)
                print(f"Received message: {msg}")
            except asyncio.TimeoutError:
                print("No message received in 10 seconds.")
            
            # Try sending a subscribe message if no welcome message
            # This is a guess; usually we need to subscribe to a symbol
            # subscribe_msg = {"action": "subscribe", "symbol": "THYAO"}
            # await websocket.send(json.dumps(subscribe_msg))
            
    except Exception as e:
        print(f"Connection failed: {e}")

if __name__ == "__main__":
    # Install websockets if not present: pip install websockets
    asyncio.run(test_connection())

File: wss_test.js
const WebSocket = require('ws');

const wsUrl = 'wss://ws.7k2v9x1r0z8t4m3n5p7w.com';

console.log(`Connecting to ${wsUrl}...`);

const ws = new WebSocket(wsUrl);

ws.on('open', function open() {
    console.log('Connected!');

    // Wait to see if we get a welcome message
    // If not, we might need to send something.
    setTimeout(() => {
        console.log('Connection open for 5 seconds.');
        ws.close();
    }, 5000);
});

ws.on('message', function incoming(data) {
    console.log('Received message:', data.toString());
});

ws.on('error', function error(err) {
    console.error('WebSocket error:', err);
});

ws.on('close', function close() {
    console.log('Disconnected');
});
